api/app/controllers/groups_controller.rb

```diff
@@ -64,6 +64,10 @@ def destroy
     end
   end
 
+  def members
+    render json: UserSerializer.new(@group.users).serializable_hash
```

Comment:
Actualmente esto retorna a los users dentro de `data` y con información un poco duplicada como el `id`:
<img width="605" alt="Screenshot 2023-10-01 at 11 41 30 PM" src="https://github.com/wyeworks/finder/assets/62676004/655ebe8d-6ed2-4a91-b497-88febd2b51fd">

Que te parece moverlo a algo como:
```suggestion
    users = @group.users.map do |user|
      UserSerializer.new(user).serializable_hash[:data][:attributes]
    end

    render json: users
```

Va a terminar retornando algo así:
<img width="451" alt="Screenshot 2023-10-01 at 11 41 47 PM" src="https://github.com/wyeworks/finder/assets/62676004/87959973-6a68-40e0-a39b-0e6353ddbe01">

Que te parece?